genesis/beacons.go

```diff
@@ -70,56 +23,9 @@ func getIPs(networkID uint32) []string {
 // getNodeIDs returns the beacon node IDs for each network
 func getNodeIDs(networkID uint32) []string {
 	switch networkID {
-	case constants.MainnetID:
-		return []string{
-			"NodeID-A6onFGyJjA37EZ7kYHANMR1PFRT8NmXrF",
-			"NodeID-6SwnPJLH8cWfrJ162JjZekbmzaFpjPcf",
-			"NodeID-GSgaA47umS1px2ohVjodW9621Ks63xDxD",
-			"NodeID-BQEo5Fy1FRKLbX51ejqDd14cuSXJKArH2",
-			"NodeID-Drv1Qh7iJvW3zGBBeRnYfCzk56VCRM2GQ",
-			"NodeID-DAtCoXfLT6Y83dgJ7FmQg8eR53hz37J79",
-			"NodeID-FGRoKnyYKFWYFMb6Xbocf4hKuyCBENgWM",
-			"NodeID-Dw7tuwxpAmcpvVGp9JzaHAR3REPoJ8f2R",
-			"NodeID-4kCLS16Wy73nt1Zm54jFZsL7Msrv3UCeJ",
-			"NodeID-9T7NXBFpp8LWCyc58YdKNoowDipdVKAWz",
-			"NodeID-6ghBh6yof5ouMCya2n9fHzhpWouiZFVVj",
-			"NodeID-HiFv1DpKXkAAfJ1NHWVqQoojjznibZXHP",
-			"NodeID-Fv3t2shrpkmvLnvNzcv1rqRKbDAYFnUor",
-			"NodeID-AaxT2P4uuPAHb7vAD8mNvjQ3jgyaV7tu9",
-			"NodeID-kZNuQMHhydefgnwjYX1fhHMpRNAs9my1",
-			"NodeID-A7GwTSd47AcDVqpTVj7YtxtjHREM33EJw",
-			"NodeID-Hr78Fy8uDYiRYocRYHXp4eLCYeb8x5UuM",
-			"NodeID-9CkG9MBNavnw7EVSRsuFr7ws9gascDQy3",
-			"NodeID-A8jypu63CWp76STwKdqP6e9hjL675kdiG",
-			"NodeID-HsBEx3L71EHWSXaE6gvk2VsNntFEZsxqc",
-			"NodeID-Nr584bLpGgbCUbZFSBaBz3Xum5wpca9Ym",
-			"NodeID-QKGoUvqcgormCoMj6yPw9isY7DX9H4mdd",
-			"NodeID-HCw7S2TVbFPDWNBo1GnFWqJ47f9rDJtt1",
-			"NodeID-FYv1Lb29SqMpywYXH7yNkcFAzRF2jvm3K",
-		}
-	case constants.FujiID:
+	case constants.ColumbusID:
 		return []string{
-			"NodeID-2m38qc95mhHXtrhjyGbe7r2NhniqHHJRB",
-			"NodeID-JjvzhxnLHLUQ5HjVRkvG827ivbLXPwA9u",
-			"NodeID-LegbVf6qaMKcsXPnLStkdc1JVktmmiDxy",
-			"NodeID-HGZ8ae74J3odT8ESreAdCtdnvWG1J4X5n",
-			"NodeID-CYKruAjwH1BmV3m37sXNuprbr7dGQuJwG",
-			"NodeID-4KXitMCoE9p2BHA6VzXtaTxLoEjNDo2Pt",
-			"NodeID-LQwRLm4cbJ7T2kxcxp4uXCU5XD8DFrE1C",
-			"NodeID-4CWTbdvgXHY1CLXqQNAp22nJDo5nAmts6",
-			"NodeID-4QBwET5o8kUhvt9xArhir4d3R25CtmZho",
-			"NodeID-JyE4P8f4cTryNV8DCz2M81bMtGhFFHexG",
-			"NodeID-EDESh4DfZFC15i613pMtWniQ9arbBZRnL",
-			"NodeID-BFa1padLXBj7VHa2JYvYGzcTBPQGjPhUy",
-			"NodeID-CZmZ9xpCzkWqjAyS7L4htzh5Lg6kf1k18",
-			"NodeID-FesGqwKq7z5nPFHa5iwZctHE5EZV9Lpdq",
-			"NodeID-84KbQHSDnojroCVY7vQ7u9Tx7pUonPaS",
-			"NodeID-CTtkcXvVdhpNp6f97LEUXPwsRD3A2ZHqP",
-			"NodeID-hArafGhY2HFTbwaaVh1CSCUCUCiJ2Vfb",
-			"NodeID-4B4rc5vdD1758JSBYL1xyvE5NHGzz6xzH",
-			"NodeID-EzGaipqomyK9UKx9DBHV6Ky3y68hoknrF",
-			"NodeID-NpagUxt6KQiwPch9Sd4osv8kD1TZnkjdk",
-			"NodeID-3VWnZNViBP2b56QBY7pNJSLzN2rkTyqnK",
+			"NodeID-PGHYeLVkU6ZVQEu8CuRBk6pQ2NJNAuzZ4",
```

Comment:
The artificial split between IP and NodeID feels somewhat unclean we have an fixed relation between the two lists of entries by their index but the lists are stored in two different places. Coming from C/C++ this should be a list of objects (either as struct or class) simply adressed by the index where the IP and NodeID is directly bound together as members of a single object.
Would be something like this possible in this context?

Reply:
Jep, agree